packages/bff/src/model/domain/errors.ts

```diff
@@ -2,38 +2,43 @@ import { constants } from "http2";
 import {
   ApiError,
   AttributeId,
+  parseErrorMessage,
   makeApiProblemBuilder,
 } from "pagopa-interop-models";
 
 export const errorCodes = {
```

Comment:
To reduce the diff and future conflicts, could you add all and only the new errors as the last ones, without changing the codes of the other errors already defined?